dhis-2/dhis-api/src/main/java/org/hisp/dhis/keyjsonvalue/KeyJsonValue.java

```diff
@@ -57,7 +57,7 @@
     /**
      * Whether or not this KeyJsonValue is encrypted or not. Default is false.
      */
-    private Boolean encrypted = false;
```

Comment:
As far as I can tell this must be non-null. While the DB allows nulls this being null would always have caused NPE when accessing the value so I figure there cannot be values in the DB where this is null.

This should make sure we are sure of that in a more well defined way